### PR TITLE
fix(blocks): database multi-select tab close icon invisible

### LINKED ISSUE #2887: The Close Icon of Database Multi-Select Tab is Invisible.
database-multi-select-tab-close-icon-invisible

https://github.com/toeverything/blocksuite/assets/99816898/4d4ca36f-95cc-4ff9-b8d9-5e040e5280b3

### PR DESCRIPTION
**Pull request by donteatfriedrice**
To fix #2887 

https://github.com/toeverything/blocksuite/assets/99816898/c612b368-6e55-4d86-9b82-ec501b8d4599

### CODE REVIEW

File: packages/global/src/config/icons/database.ts [hunk 1]
```diff
@@ -259,7 +259,6 @@ export const DatabaseSearchClose = html`<svg
   width="16"
   height="16"
   viewBox="0 0 16 16"
-  fill="none"
```
Comment (zqran) [inline packages/global/src/config/icons/database.ts:262]: We can keep `fill="none"` here.

File: packages/blocks/src/database-block/table/components/column-type/select/select-cell-editing.ts [hunk 2]
```diff
@@ -97,6 +97,12 @@ const styles = css`
   .select-selected > .close-icon {
     display: flex;
     align-items: center;
+    fill: var(--affine-icon-color);
```
Comment (zqran) [inline packages/blocks/src/database-block/table/components/column-type/select/select-cell-editing.ts:100]: In our design system, the icon color here should be: 

```suggestion
    fill: var(--affine-black-90);
```

File: packages/blocks/src/database-block/table/components/column-type/select/select-cell-editing.ts [hunk 3]
```diff
@@ -97,6 +97,12 @@ const styles = css`
...
+  }
+
+  .select-selected > .close-icon:hover {
+    cursor: pointer;
+    background: var(--affine-hover-color);
```
Comment (zqran) [inline packages/blocks/src/database-block/table/components/column-type/select/select-cell-editing.ts:105]: The hover `background` is not needed here, this is also by design.

File: packages/blocks/src/database-block/table/components/column-type/select/select-cell-editing.ts (hunk 2 above)
Comment (zqran) [inline packages/blocks/src/database-block/table/components/column-type/select/select-cell-editing.ts:100]: In our design system, the icon color here should be:

```suggestion
    fill: var(--affine-black-90);
```

File: packages/blocks/src/database-block/table/components/column-type/select/select-cell-editing.ts (hunk 3 above)
Comment (zqran) [inline packages/blocks/src/database-block/table/components/column-type/select/select-cell-editing.ts:105]: The hover `background` is not needed here, this is also by design.

File: packages/global/src/config/icons/database.ts (hunk 1 above)
Comment (zqran) [inline packages/global/src/config/icons/database.ts:262]: We can keep `fill="none"` here.

**Review by zqran — DISMISSED** [review]

File: packages/blocks/src/database-block/table/components/column-type/select/select-cell-editing.ts (hunk 2 above)
Comment (donteatfriedrice) [reply packages/blocks/src/database-block/table/components/column-type/select/select-cell-editing.ts:100]: Thanks.

File: packages/blocks/src/database-block/table/components/column-type/select/select-cell-editing.ts (hunk 3 above)
Comment (donteatfriedrice) [reply packages/blocks/src/database-block/table/components/column-type/select/select-cell-editing.ts:105]: Thanks again. By the way, should I keep the cursor: pointer or not?

File: packages/global/src/config/icons/database.ts (hunk 1 above)
Comment (donteatfriedrice) [reply packages/global/src/config/icons/database.ts:262]: It seems when keep fill="none" here, and add  fill: var(--affine-black-90) in .close-icon, the icon will be still not visible. Any suggestion to fix it?

File: packages/blocks/src/database-block/table/components/column-type/select/select-cell-editing.ts (hunk 3 above)
Comment (zqran) [reply packages/blocks/src/database-block/table/components/column-type/select/select-cell-editing.ts:105]: > Thanks again. By the way, should I keep the cursor: pointer or not?

Yes. Generally speaking, the clickable area should be indicated by the mouse style(`cursor: pointer`).

File: packages/global/src/config/icons/database.ts (hunk 1 above)
Comment (zqran) [reply packages/global/src/config/icons/database.ts:262]: That's fine too. Alternatively, you can set the fill style like `.close-icon > svg`.

Comment (donteatfriedrice) [reply packages/global/src/config/icons/database.ts:262]: OK. Thanks for your advice.

File: packages/blocks/src/database-block/table/components/column-type/select/select-cell-editing.ts (hunk 3 above)
Comment (donteatfriedrice) [reply packages/blocks/src/database-block/table/components/column-type/select/select-cell-editing.ts:105]: Thanks

**Review by doodlewind — APPROVED** [review]

**Pull request merged**